projects/plugins/jetpack/modules/sharedaddy/services/class-jetpack-mastodon-modal.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * Display a Mastodon modal, where folks can enter a Mastodon instance.
+ *
+ * @package automattic/jetpack
+ */
+
+/**
+ * Class Mastodon_Modal
+ */
+class Jetpack_Mastodon_Modal {
+	/**
+	 * Mastodon SVG icon.
+	 *
+	 * @var string
+	 */
+	private static $svg = '<svg width="261" height="66" viewBox="0 0 261 66" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Would it make sense to put this image in the images directory, and read it with `file_get_contents()` if necessary instead of embedding it in the PHP file?

Reply:
Right now I don't really imagine that logo will be used anywhere else, so I'm okay with hardcoding it in here. If we ever start using a full logo like that, I'll definitely move it out to its own file.

Reply:
I was thinking more that it would be easier to deal with as an actual SVG file than as a long PHP string.